catalog/schematron/catalog-schematron-plugin/src/test/java/ddf/services/schematron/SchematronValidationServiceTest.java

```diff
@@ -121,11 +145,11 @@ private SchematronValidationService getService(boolean suppressWarnings, String
 
         ArrayList<String> schemaFiles = new ArrayList<>();
         for (String schematronFile : schematronFiles) {
+            String path = schematronFile;
             if (useClassLoader) {
```

Comment:
I wanted to be able to test both a file that does not exist and a file with spaces in the name. If I use the classloader on these files to get the path, it will blow up trying to resolve rather than exercise the error checking I have in my code. I could pull it out somewhere, but I thought having one getService call made all the tests a bit cleaner.